### ISSUE TITLE
vty-5.35 failed to build in Stackage Nightly

### ISSUE THREAD

**Issue by juhp** [jtdaugherty/vty#240]
```
/var/stackage/work/unpack-dir/unpacked/vty-5.35-c58fd74e06bfb90fba769275b77248248bf1dd450504f74a80ad22e93a4804e6/test/VerifyUsingMockInput.hs:139:44: error:
    • Couldn't match type ‘InternalEvent’ with ‘Event’
      Expected: [Event]
        Actual: [InternalEvent]
    • In the third argument of ‘compareEvents’, namely ‘outEvents’
      In a stmt of a 'do' block:
        compareEvents inputSpec expectedEvents outEvents
      In the expression:
        do let maxDuration = sum ... + minDetectableDelay
               eventCount = length expectedEvents
           (writeFd, readFd) <- openPseudoTerminal
           (setTermAttr, _) <- attributeControl readFd
           setTermAttr
           ....
    |
139 |     compareEvents inputSpec expectedEvents outEvents
    |                                            ^^^^^^^^^
```

**Comment by jtdaugherty** [comment]
Thanks for this report! This is now fixed in 5.35.1.

**Comment by juhp** [comment]
Thanks so fast
